core/src/main/java/org/apache/calcite/sql/SqlDateTimeFormat.java

```diff
@@ -44,6 +44,12 @@
   ABBREVIATEDDAYOFWEEK("EEE"),
   TWENTYFOURHOUR("HH24"),
   HOUR("HH"),
+  TWENTYFOURHOURMIN("HH24MI"),
+  TWENTYFOURHOURMINSEC("HH24MISS"),
+  YYYYMMDDHH24MISS("YYYYMMDDHH24MISS"),
```

Comment:
@soma-mondal @praveendatametica do we plan to keep adding formats to the standard as when they come in client code

Reply:
We need to  keep standard in calcite so that for others they can map the same

Reply:
butl ets take this entry for example YYYYMMDDHH24MISS ... this is a combination of YYYY, MM, DD, HH24, MI and SS components in netezza. All of these components already have representation in SqlDateTimeFormat. So instead of adding a new entry in standard why cant we do this in MIg , recreate this format by mapping each component of format to its standard and concatenating it.

Reply:
Please Refer to HiveDateFormat for such examples ... will merge it tomoorow